airflow/bin/cli.py

```diff
@@ -185,6 +185,56 @@ def trigger_dag(args):
     logging.info(message)
 
 
+def batch_set_pools(args):
+    """
+    set a batch of Airflow pools
+
+    :param args: arguments from argparser
+    :return:
+    """
+    session = settings.Session()
+    pool_names = args.names
+    pool_slots = args.slots
+    pool_descriptions = args.descriptions
+    print("Number of pool names input: {}".format(len(pool_names)))
+    print("Number of pool slots count input: {}".format(len(pool_slots)))
+    print("Number of pool description input: {}".format(len(pool_descriptions)))
+
+    if len(pool_names) != len(pool_slots) or len(pool_names) != len(pool_descriptions):
```

Comment:
So if we have 50 pools, we'll have 3 lists of arguments, each of which must be exactly 50 elements long?

Would it make sense to read these values from some sort of config file (i.e, ini, yaml, json, .csv), etc. than from the command line?

Maybe @mistercrunch would have a recommendation on how to design this interface.

Reply:
Sure. I feel we don't want to put the airflow_pools into incubator-airflow repo data-platform-upgrade branch as it will be more troublesome for our users to add or change a pool. We can put it into a config file in etl repo and when `airflow batch_set_pools` will try to import file when it is being called, but this also means the change of this PR will not benefit other cases because we need to somehow hardcode the config file path in incubator-airflow data-platform-upgrade branch?

Reply:
+1 on reading from a config instead of passing a long list.